Amplify/Categories/DataStore/Model/Collection/List+Model.swift

```diff
@@ -123,14 +130,15 @@ open class List<ModelType: Model>: ModelList {
 
     // MARK: Combine
 
-    /// Lazy load the collection and expose the loaded `Elements` as a Combine `Publisher`.
-    /// This is useful for integrating the `List<ModelType>` with existing Combine code
-    /// and/or SwiftUI.
+    /// Load data into the collection from the data source and expose the data as a Combine `Publisher`. This is useful
+    /// for integrating the `List<ModelType>` with existing Combine code and/or SwiftUI. This method has been deprecated
+    /// and is currently not a supported API.
```

Comment:
Update docs and deprecation message to note that it will always crash with a fatal error

Reply:
going over the latest comments to apply to latest PR; the fatal error is no longer relevant since List is no longer an "abstract" class, the deprecation remains